docs/SupportedCommands/Commands.md

```diff
@@ -0,0 +1,70 @@
+## Supported Commands
+
+|          Command           |          Engine Version Support          |
```

Comment:
'move', 'longclick', 'down', 'up' are missing and available in 4.2.7+.
We need to find the correct names